app/upgrades/v5/upgrades.go

```diff
@@ -99,3 +106,50 @@ func MigrateGenesis(appState genutiltypes.AppMap, clientCtx client.Context) genu
 
 	return appState
 }
+
+// ResolveAirdrop will iterate over all the available claim records and
```

Comment:
would be nice to describe the motivation for this change too, i.e why are we updating claims records?